### PR TITLE
Docs 7209 partner manager 2 x install steps mc

### PR DESCRIPTION
**Pull request by mcherem**

### CODE REVIEW

File: modules/ROOT/pages/setup.adoc [hunk 1]
```diff
@@ -79,8 +83,12 @@ wrapper.java.maxmemory=4606
...
 
+. Install your license. See https://docs.mulesoft.com/mule-runtime/4.2/installing-an-enterprise-license[Install an Enterprise License] for more information.
+
 . Add the server in Runtime Manager based on the Runtime manager https://docs.mulesoft.com/runtime-manager/servers-create#create-a-server[Create a Server].
 
+. Start the Mule runtime and verify it is active within Anypoint Runtime Manager.
+
 . Include the created servers into the server group you created earlier: *B2BServerGroup*: https://docs.mulesoft.com/runtime-manager/server-group-add[Add Servers to a Group]
 
 . Start the Mule runtime and verify it is active within Anypoint Runtime Manager.
```
Comment (vijayang) [inline modules/ROOT/pages/setup.adoc]:
```suggestion
. Start the Mule runtime and verify it is active within Anypoint Runtime Manager.
```
Start the Mule runtime and verify it is shown as Running in Anypoint Runtime Manager.

**Review by vijayang — COMMENTED** [review]
For each environment, set up your Runtime by https://docs.mulesoft.com/mule-runtime/4.2/runtime-installation-task[Downloading the MuleSoft runtime] and unzipping it.
^^ This title needs to be changed to "Download and Install Mule"

**Review by vijayang — COMMENTED** [review]
For each environment, set up your Runtime by https://docs.mulesoft.com/mule-runtime/4.2/runtime-installation-task[Downloading the MuleSoft runtime] and unzipping it.
^^ This title needs to be changed to "Download and Install Mule"

Line# 91: Change "establish an API" to "implement an API"

File: modules/ROOT/pages/setup.adoc (hunk 1 above)
Comment (mcherem) [reply modules/ROOT/pages/setup.adoc]: Fixed.

**Review by mcherem — COMMENTED** [review]
Addressed all Vijay's comments.

File: modules/ROOT/pages/setup.adoc [hunk 2]
```diff
@@ -25,10 +25,14 @@ Roles are allocated via the Access Management section in Anypoint Platform.
 
 === Partner Manager related technical requirements:
 
-* Mule runtime 4.2.2 or higher. Make sure you have the required Mule runtime capacity to handle your B2B transaction workload needs.
-If you are unsure about the Mule runtime capacity required to handle your B2B transactions, contact your MuleSoft account representative for assistance.
-
-* A storage solution to store the B2B transmission payloads you send or receive. Storage options can include a Database, Amazon S3, Azure Blob Storage, and so on.
+* Mule runtime 4.2.2 or higher.
```
Comment (kbostrom) [inline modules/ROOT/pages/setup.adoc]: GLOBALLY REPLACE any other variation of "Mule runtime engine" or "Mule" with one of those (only two) accepted presentations. Doing so should be muscle-memory by now :-)

Comment (kbostrom) [reply modules/ROOT/pages/setup.adoc]:
```suggestion
* Mule runtime engine (Mule) 4.2.2 or later
```
Not "higher": "later." See style ref.
No period.

File: modules/ROOT/pages/setup.adoc [hunk 3]
```diff
@@ -25,10 +25,14 @@ Roles are allocated via the Access Management section in Anypoint Platform.
...
++
+Make sure you have the required Mule runtime capacity to handle your B2B transaction workload needs. If you are unsure about the Mule runtime capacity required to handle your B2B transactions, contact your MuleSoft account representative for assistance.
```
Comment (kbostrom) [inline modules/ROOT/pages/setup.adoc]:
```suggestion
Ensure that you have the required Mule capacity to handle your B2B transaction workload needs. If you are unsure what that capacity is, contact your MuleSoft account representative for assistance.
```

File: modules/ROOT/pages/setup.adoc [hunk 4]
```diff
@@ -25,10 +25,14 @@ Roles are allocated via the Access Management section in Anypoint Platform.
...
++
+* A storage solution to store the B2B transmission payloads you send or receive.
```
Comment (kbostrom) [inline modules/ROOT/pages/setup.adoc]:
```suggestion
* A storage solution for the B2B transmission payloads you send or receive
```

File: modules/ROOT/pages/setup.adoc [hunk 5]
```diff
@@ -25,10 +25,14 @@ Roles are allocated via the Access Management section in Anypoint Platform.
...
++
+Storage options can include a Database, Amazon S3, Azure Blob Storage, and so on.
```
Comment (kbostrom) [inline modules/ROOT/pages/setup.adoc]:
```suggestion
Storage options can include a database, Amazon S3, Azure Blob Storage, and so on.
```

File: modules/ROOT/pages/setup.adoc [hunk 6]
```diff
@@ -25,10 +25,14 @@ Roles are allocated via the Access Management section in Anypoint Platform.
...
++
 
 * Anypoint Studio 7.4 or higher.
```
Comment (kbostrom) [inline modules/ROOT/pages/setup.adoc]:
```suggestion
* Anypoint Studio 7.4 or later
```

File: modules/ROOT/pages/setup.adoc [hunk 7]
```diff
@@ -58,7 +62,7 @@ All B2B processing Mule apps are deployed to this server group. The steps to set
 IMPORTANT: The server group you setup must use the exact name: *B2BServerGroup*. This name is case sensitive.
 +
 
-. For each environment, set up your Runtime by https://docs.mulesoft.com/mule-runtime/4.2/runtime-installation-task[Downloading the MuleSoft runtime] and unzipping it.
+. For each environment, set up your Runtime by https://docs.mulesoft.com/mule-runtime/4.2/runtime-installation-task[Download and Install Mule] and unzipping it.
```
Comment (kbostrom) [inline modules/ROOT/pages/setup.adoc]:
```suggestion
. For each environment, set up your Mule instance by https://docs.mulesoft.com/mule-runtime/4.2/runtime-installation-task[downloading and installing Mule] and unzipping it.
```

File: modules/ROOT/pages/setup.adoc [hunk 8]
```diff
@@ -79,16 +83,20 @@ wrapper.java.maxmemory=4606
 
 ----
 
+. Install your license. See https://docs.mulesoft.com/mule-runtime/4.2/installing-an-enterprise-license[Install an Enterprise License] for more information.
```
Comment (kbostrom) [inline modules/ROOT/pages/setup.adoc]: This should also be muscle memory by now: limit steps and list items to no more than one sentence.
```suggestion
. Install your license. 
See https://docs.mulesoft.com/mule-runtime/4.2/installing-an-enterprise-license[Install an Enterprise License] for more information.
```

File: modules/ROOT/pages/setup.adoc [hunk 9]
```diff
@@ -79,16 +83,20 @@ wrapper.java.maxmemory=4606
...
+
 . Add the server in Runtime Manager based on the Runtime manager https://docs.mulesoft.com/runtime-manager/servers-create#create-a-server[Create a Server].
```
Comment (kbostrom) [inline modules/ROOT/pages/setup.adoc]:
```suggestion
. Add the server in Runtime Manager based on the instructions in https://docs.mulesoft.com/runtime-manager/servers-create#create-a-server[Create a Server].
```

File: modules/ROOT/pages/setup.adoc [hunk 10]
```diff
@@ -79,16 +83,20 @@ wrapper.java.maxmemory=4606
...
 
+. Start the Mule runtime and verify it is active within Anypoint Runtime Manager.
```
Comment (kbostrom) [inline modules/ROOT/pages/setup.adoc]:
```suggestion
. Start Mule and verify it is active within Anypoint Runtime Manager.
```

File: modules/ROOT/pages/setup.adoc [hunk 11]
```diff
@@ -79,16 +83,20 @@ wrapper.java.maxmemory=4606
...
+
 . Include the created servers into the server group you created earlier: *B2BServerGroup*: https://docs.mulesoft.com/runtime-manager/server-group-add[Add Servers to a Group]
 
-. Start the Mule runtime and verify it is active within Anypoint Runtime Manager.
+. Start the Mule runtime and verify it shows as 'Running' in Anypoint Runtime Manager.
```
Comment (kbostrom) [inline modules/ROOT/pages/setup.adoc]: Why do we repeat this step? This looks like a mistake.
```suggestion
. Start Mule and verify that it shows as 'Running' in Anypoint Runtime Manager.
```

**Review by kbostrom — APPROVED** [review]
Done!
Many of these edits are issues that I have commented on before. Please create and post a list for yourself of the style recommendations that you find difficult to remember, such as the proper way to refer to Mule and limiting steps and list items to one sentence.

File: modules/ROOT/pages/setup.adoc (hunk 2 above)
Comment (mcherem) [reply modules/ROOT/pages/setup.adoc]: Fixed.

File: modules/ROOT/pages/setup.adoc (hunk 3 above)
Comment (mcherem) [reply modules/ROOT/pages/setup.adoc]: Fixed.

File: modules/ROOT/pages/setup.adoc (hunk 4 above)
Comment (mcherem) [reply modules/ROOT/pages/setup.adoc]: Fixed.

File: modules/ROOT/pages/setup.adoc (hunk 5 above)
Comment (mcherem) [reply modules/ROOT/pages/setup.adoc]: Fixed.

File: modules/ROOT/pages/setup.adoc (hunk 6 above)
Comment (mcherem) [reply modules/ROOT/pages/setup.adoc]: Fixed.

File: modules/ROOT/pages/setup.adoc (hunk 7 above)
Comment (mcherem) [reply modules/ROOT/pages/setup.adoc]: Fixed. The second one is  the title of the doc, so won't change.

File: modules/ROOT/pages/setup.adoc (hunk 8 above)
Comment (mcherem) [reply modules/ROOT/pages/setup.adoc]: Fixed.

File: modules/ROOT/pages/setup.adoc (hunk 9 above)
Comment (mcherem) [reply modules/ROOT/pages/setup.adoc]: Fixed.

File: modules/ROOT/pages/setup.adoc (hunk 10 above)
Comment (mcherem) [reply modules/ROOT/pages/setup.adoc]: Fixed.

File: modules/ROOT/pages/setup.adoc (hunk 11 above)
Comment (mcherem) [reply modules/ROOT/pages/setup.adoc]: Content is changed.

**Pull request merged**